integration/mocks_test.go

```diff
@@ -190,9 +190,4 @@ func TestMocks(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	mockCtx.AssertExpectations(t)
-	mockChildHandle.AssertExpectations(t)
-	mockGenericHandle.AssertExpectations(t)
```

Comment:
A mock built with `mocks.NewMoc...` already checks expectations before destroying the object.